stdlib/shlex.pyi

```diff
@@ -27,7 +27,7 @@ def join(split_command: Iterable[str]) -> str: ...
 def quote(s: str) -> str: ...
 
 # TODO: Make generic over infile once PEP 696 is implemented.
-class shlex(Iterable[str]):
+class shlex(Iterator[str]):
```

Comment:
```suggestion
class shlex:
```
Since Iterable/Iterator are protocols, this should be fine. At runtime this class doesn't have any base classes, so removing the base classes better reflects the runtime.

Not sure what I was thinking in 2018 when I changed the base class here.